### PR TITLE
Add io.github.achetagames.epic_asset_manager

### PR DESCRIPTION
**Pull request by aknarts**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

There are two non standard permissions:
--talk-name=org.freedesktop.secrets - needed to access secrets used by the application
--filesystem=host - Sadly needed as this is a downloader tool that drops assets to a user specified location and eventually to user projects that can be placed anywhere on the machine

### CODE REVIEW

**Comment by aknarts** [comment]
bot, build com.belmoussaoui.Authenticator

**Comment by aknarts** [comment]
bot, build io.github.achetagames.epic_asset_manager

**Comment by aknarts** [comment]
> Queued test build for com.belmoussaoui.Authenticator.

Used wrong name...:/

**Comment by aknarts** [comment]
bot, build io.github.achetagames.epic_asset_manager

**Comment by aknarts** [comment]
bot, build io.github.achetagames.epic_asset_manager

**Comment by aknarts** [comment]
bot, build io.github.achetagames.epic_asset_manager

**Comment by aknarts** [comment]
bot, build io.github.achetagames.epic_asset_manager

**Comment by aknarts** [comment]
bot, build io.github.achetagames.epic_asset_manager

File: io.github.achetagames.epic_asset_manager.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      ],
+      "cleanup" : [
+        "/include",
+        "/lib/pkgconfig"
+      ],
+      "sources" : [
+        {
+          "type" : "git",
+          "url" : "https://gitlab.gnome.org/GNOME/libadwaita.git",
+          "branch" : "main",
```
Comment (hfiguiere) [inline io.github.achetagames.epic_asset_manager.json:51]:
```suggestion
```

A branch move which mean next commit, the build break. Build should be reproducible.

File: io.github.achetagames.epic_asset_manager.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      ],
+      "modules" : [
+        {
+          "name" : "libsass",
+          "buildsystem" : "meson",
+          "sources" : [
+            {
+              "type" : "git",
+              "url" : "https://github.com/lazka/libsass.git",
+              "branch" : "meson",
```
Comment (hfiguiere) [inline io.github.achetagames.epic_asset_manager.json:63]:
```suggestion
```

File: io.github.achetagames.epic_asset_manager.json [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+          ]
+        },
+        {
+          "name" : "sassc",
+          "buildsystem" : "meson",
+          "sources" : [
+            {
+              "type" : "git",
+              "url" : "https://github.com/lazka/sassc.git",
+              "branch" : "meson",
```
Comment (hfiguiere) [inline io.github.achetagames.epic_asset_manager.json:75]:
```suggestion
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: io.github.achetagames.epic_asset_manager.json [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin",
+    "build-args": [
+      "--share=network"
+    ],
+    "test-args": [
+      "--socket=x11",
+      "--share=network"
+    ],
```
[the remark this answers is not in the document]
Comment (hfiguiere) [reply io.github.achetagames.epic_asset_manager.json:27]: For the record, the last build failure says:
`flatpak-builder: Cleanup libsassError: Can't specify build-args in sandboxed build`

[the remark this answers is not in the document]
Comment (aknarts) [reply io.github.achetagames.epic_asset_manager.json:27]: That might be the reason why I am failing the build... thanks for pointing those out.

File: io.github.achetagames.epic_asset_manager.json [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+  "sdk": "org.gnome.Sdk",
+  "command": "epic_asset_manager",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable"
+  ],
+  "finish-args": [
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--share=network",
+    "--filesystem=host",
```
[the remark this answers is not in the document]
Comment (aknarts) [reply io.github.achetagames.epic_asset_manager.json:14]: Explained in the PR, it is a downloader where user specifies where they want to download and eventually place the assets into their projects. From what research I did from people who do use the tool already it is in very random locations. That is why, sadly...

**Comment by aknarts** [comment]
bot, build io.github.achetagames.epic_asset_manager

**Comment by aknarts** [comment]
Sorry for the mess I made trying to figure this one out, it is my first one, it was a learning experience.

**Pull request closed without merging**